src/spell.js

```diff
@@ -81,9 +81,11 @@ function parseObjectValue(value) {
  * @returns {boolean}
  */
 function isOperatorCondition(condition) {
+  const keys = Object.keys(condition);
```

Comment:
`Object.keys(condition)` should be called after `isPlainObject(condition)` passes

Reply:
which is the reason that causes tests to fail